packages/plans-grid-next/src/components/actions.tsx

```diff
@@ -454,59 +354,69 @@ const PlanFeaturesActionsButton: React.FC< PlanFeaturesActionsButtonProps > = (
 		storageAddOnsForPlan,
 	} );
 
-	const onCtaClick = useActionCallback( {
+	const priceString = formatCurrency(
```

Comment:
Note:

`priceString` already existed. I moved its declaration earlier in the file.